.formatter.exs

```diff
@@ -1,6 +1,30 @@
+locals_without_parens = [
+  # Query
+  from: 2,
+
+  # Schema
+  field: 1,
```

Comment:
Should `schema`, `embedded_schema` and `timestamps` be also be included? The doc examples have all of them without parens https://hexdocs.pm/ecto/Ecto.Schema.html#content

Reply:
We don't need schema and embedded_schema because they are used in do-blocks and those don't have parens by default. We should add timestamps/1 though. Good catch!

Reply:
After Elixir 1.5, functions without arity is required to have parens. For `timestamps` I think if we need parens "sometimes", it's not consistent and I prefer always adding parens. That's the reason I didn't added it originally.

One example:

```elixir
# must have parens
timestamps()

# no parens
timestamps inserted_at: :created_at
```

What do you think?